examples/infer/main.go

```diff
@@ -20,7 +20,7 @@ func main() {
 	}
 	fmt.Println("## Raw Table ##")
 	fmt.Println(tab)
-	sch, err := schema.Infer(tab)
+	sch, err := schema.Infer(tab, schema.SampleLimit(-1))
```

Comment:
Would you mind creating a public constant that represents this -1. Somenhing like schema.SampleAllRows